eppoclient/initclient.go

```diff
@@ -13,7 +13,7 @@ func InitClient(config Config) *EppoClient {
 	sdkParams := SDKParams{apiKey: config.ApiKey, sdkName: "go", sdkVersion: __version__}
 
 	httpClient := newHttpClient(config.BaseUrl, &http.Client{Timeout: REQUEST_TIMEOUT_SECONDS}, sdkParams)
-	configStore := newConfigurationStore(MAX_CACHE_ENTRIES)
+	configStore := newConfigurationStore()
```

Comment:
why was this removed?

Reply:
The cache is used incorrectly in the SDKs. It does not speed up things but rather adds overhead (to track which entries were used last).